metrics/confusion_matrix/README.md

```diff
@@ -79,6 +77,16 @@ Example 1 - A simple example
 {'confusion_matrix': [[2, 0, 0], [0, 1, 1], [1, 1, 1]]}
 ```
 
+Example 2 - Multilabel scenario with binary labels
+
+```python
+>>> confusion_metric = evaluate.load("confusion_matrix")
+>>> confusion_metric.config_name = "multilabel"
```

Comment:
I think it would be cleaner to directly pass the config in the load method.

Reply:
Done ✅